ercc/main.go

```diff
@@ -27,6 +27,7 @@ func main() {
 	c := &registry.Contract{}
 	c.Verifier = attestation.NewVerifier()
 	c.IEvaluator = &utils.IdentityEvaluator{}
+	c.BeforeTransaction = registry.MyBeforeTransaction
```

Comment:
can't see this used anywhere, nor defined

Reply:
It is defined in `ercc/registry/registry.go` for logging.